capstone-backend/src/main/java/com/google/sps/servlets/Utility.java

```diff
@@ -26,6 +27,7 @@
 import java.io.IOException;
 import java.lang.reflect.Field;
 import java.util.ArrayList;
+import java.util.HashMap;
```

Comment:
nit: unused imports

import com.google.cloud.firestore.QueryDocumentSnapshot;
import com.google.gson.Gson;
import com.google.gson.JsonElement;
import java.io.BufferedReader;
import java.io.InputStreamReader;

Reply:
Thanks for the catch! Once these last PRs are merged, I'm going to move over to gitpod, and it'll warn me about these sorts of things so you don't have to anymore :)